contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -1511,17 +1509,22 @@ boolean CreateProcessW(String lpApplicationName, char[] lpCommandLine,
      *         {@link #GetLastError()} returns an error code other than
      *         {@link WinError#ERROR_OLD_WIN_VERSION}.
      */
+    boolean VerifyVersionInfo(OSVERSIONINFOEX lpVersionInformation, int dwTypeMask, long dwlConditionMask);
+
+    /**
+     * @deprecated Use {@link #VerifyVersionInfo(com.sun.jna.platform.win32.WinNT.OSVERSIONINFOEX, int, long)}
+     */
     boolean VerifyVersionInfoW(OSVERSIONINFOEX lpVersionInformation, int dwTypeMask, long dwlConditionMask);
```

Comment:
It's either changing the signature of VerifyVersionInfoW or GetVersionEx. If OSVERSIONINFOEX is the W version, then GetVerionEx cannot take that struct as an argument.